lib/tools/adb-commands.js

```diff
@@ -83,7 +83,15 @@ methods.getApiLevel = async function getApiLevel () {
   if (!_.isInteger(this._apiLevel)) {
     try {
       const strOutput = await this.getDeviceProperty('ro.build.version.sdk');
-      this._apiLevel = parseInt(strOutput.trim(), 10);
+      let apiLevel = parseInt(strOutput.trim(), 10);
+
+      // Temp workaround. Android Q beta emulators report SDK 28 when they should be 29
+      if (apiLevel === 28) {
+        if ((await this.getDeviceProperty('ro.build.version.release')).toLowerCase() === 'q') {
```

Comment:
no need to have a nested if here. Use &&

Reply:
Done.